### PR TITLE
🐟 Add pr for fish

### PR DESCRIPTION
**Pull request by 9at8**
Don't know why the bash script uses the ruby script :man_shrugging: 

PS: I opened this using `fk pr` :wink:

### CODE REVIEW

**Comment by arora-aman** [comment]
> Don't know why the bash script uses the ruby script 🤷‍♂️

Yeah that is not needed. It redirects to existing PR.

Created #35

File: functions/_fk_pr.fish [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+function _fk_pr
+    if ! git rev-parse --show-toplevel > /dev/null 2> /dev/null
+        echo "Cannot open PR since current directory is not a git repo"
+        return 1
+    end
+
+    set curr_branch (git symbolic-ref --short HEAD)
+
+    if test curr_branch = "master"
+        echo "Cannot open PR for master branch"
```
Comment (arora-aman) [inline functions/_fk_pr.fish]: you can in case master is not default.

File: functions/_fk_pr.fish [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        return 2
+    end
+
+    set git_remote (git config --get remote.origin.url)
+    set repo_name (echo $git_remote | xargs basename | cut -d'.' -f1)
```
Comment (arora-aman) [inline functions/_fk_pr.fish]: Will cause issues with SSH vs HTTPS remotes. Check https://github.com/fantastic-kit/fantastic-kit/pull/31/

File: functions/_fk_script.fish [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env fish
+
```
Comment (arora-aman) [inline functions/_fk_script.fish:2]: what is this and why is it needed?

**Review by arora-aman — DISMISSED** [review]

**Comment by arora-aman** [comment]
also update readme, since the other pr is probably going in soon

File: functions/_fk_script.fish (hunk 3 above)
Comment (9at8) [reply functions/_fk_script.fish:2]: This is a helper function for calling ruby scripts. This isn't needed right now but will be useful later.

File: functions/_fk_pr.fish (hunk 2 above)
Comment (9at8) [reply functions/_fk_pr.fish]: not this line, the next one.

**Comment by Azhng** [comment]
I think my original intention was to eventually able to using github GraphQL API to check if a PR exists or not. That's why there's a ruby script. At the moment this is more or less just a placeholder until I can figure out how to use Linux keyring to store user's github credentials.

**Review by Azhng — APPROVED** [review]

**Pull request merged**